src/API/VendorAPI.php

```diff
@@ -78,22 +92,48 @@ public function get_version() {
 	 * @since 1.0.0
 	 */
 	public function register_routes() {
+		if ( $this->supports_multiple_endpoints ) {
+			$this->register_router_multiple_methods();
+		} else {
+			$this->register_router_single_method();
+		}
+	}
 
+	/**
+	 * Register endpoint route with single method
+	 *
+	 * @param string $methods The endpoint's methods.
+	 * @param string $endpoint_callback The endpoint's callback.
+	 *
+	 * @since 1.0.11
```

Comment:
We have a find/replace process to add the correct version in these tags at the time of the release, so this should instead be the placeholder of `x.x.x`.

```suggestion
	 * @since x.x.x
```